src/codegen/ast_interpreter.cpp

```diff
@@ -561,6 +562,12 @@ Value ASTInterpreter::visit_slice(AST_slice* node) {
     return Value();
 }
 
+Value ASTInterpreter::visit_ellipsis(AST_Ellipsis* node) {
+    Value v;
```

Comment:
You also have to ether call ```abortJITing()``` inside this function or tell the baseline jit how to generate code for the ```ellipsis``` node. Sorry that's not really obvious :-( - our interpreter calls in the appropriate functions of the baseline jit to generate the requested code.
With your current code we would probably hit some assert if a ellipsis ast node is contained in a function which gets called more than 25 times (=baseline jit kicks in) because only the ```Box* o``` field is set but not the ```RewriterVar* var``` field of the ```Value``` object.
So we could call ```abortJITing()``` but the much better solution is to actually teach the basline jit how to it can generate machine code for this node and in this case this is also very easy :-).
```return Value(Ellipsis, jit ? jit->imm(Ellipsis) : NULL);``` should do it.